### ISSUE TITLE
Make it work like Windows

### ISSUE THREAD

**Issue by muhdsalm** [Vanilla-OS/legendary-desktop-feedback#14]
**Is your feature request related to a problem? Please describe.**
I'm always frustrated when Linux isn't exactly like Windows.

**Describe the solution you'd like**
I would like there to be a mode where it becomes Windows.

**Describe alternatives you've considered**
I've considered switching to Windows but my neighbor installed Linux and I don't know how to switch back. Smelly nerd.

**Additional context**
The desktop environment can look like MacOS if it wants but it must have a mode where it turns into Windows. It must also work like Windows. I want to play Valorant and it doesn't work on Linux and I want you to fix it right now.

**Comment by SethbotStar** [comment]
What the desktop environment may be able to do is implement an RDP to Wayland converter via the compositor, and utilize Linux docker or podman containers containing the Linux kernel as well as libvirt, and maybe qemu, as well as minimal wayland graphics with a tool to shrink down a windows ISO, and edit registry files to not show the windows version with winget and chocolatey accessible via a CLI/GUI application.

With that done we could download the NT kernel, File Explorer and the full windows software, removing the unnecessary Linux components. After that, format the drive to NTFS and install windows.

**Comment by HappiePlant** [comment]
Adding a LinkedIn shortcut is essential to make Legendary DE work like Windows. [KDE also wanted to add this but the effort got shut down by the team.](https://invent.kde.org/plasma/plasma-desktop/-/merge_requests/1731) Adding this is a HUGE opportunity for this DE. The shortcut must be <kbd>Meta</kbd> + <kbd>Ctrl</kbd> + <kbd>Alt</kbd> + <kbd>Shift</kbd> + <kbd>L</kbd> for maximum parity with Windows.

**Comment by muhdsalm** [comment]
Of course! We must also run Microsoft services in the background, to make sure we are compatible with the most privacy-respecting Operating System in the world.

**Comment by berserkwarwolf** [comment]
Please add Clippy and Xbox Game Bar.
